user_install.sh

```diff
@@ -50,6 +50,7 @@ check_host() {
 #=======================#
 
 CATKIN_DIR=~/mil_ws
+MIL_CONFIG_DIR=~/.mil
 BASHRC_FILE=~/.bashrc
 MILRC_FILE=~/.milrc
```

Comment:
Just to keep things updated on here: There was a vote in lab; the majority sided with @DSsoto.